mods/key_value.js

```diff
@@ -255,7 +266,8 @@ KVBucket.prototype.putRevision = function(restbase, req) {
                 key: rp.key,
                 tid: tid,
                 value: req.body,
-                headers: req.headers
+                headers: req.headers,
+                'content-type': req.headers && req.headers['content-type']
```

Comment:
On the previous key_value patch I've intentionally removed this line. This data is already stored in `headers`, so what's the point to duplicate it?